graspologic/models/edge_swaps.py

```diff
@@ -52,6 +56,14 @@ def __init__(self, adjacency: AdjacencyMatrix):
         direct_check = is_symmetric(adjacency)
         check_argument(direct_check, "adjacency must be undirected")
 
+        max_seed = np.iinfo(np.int32).max
```

Comment:
Consider using the unsigned integer `np.uint64` instead since you only care about the non-negatives? Theres an edge case here when someone does want to use a larger integer than a 32 bit signed integer can allow.

Reply:
so i had the same thought but for some reason windows system was puking - i had to specify a different datatype somewhere else (I think when I actually made the seeds) and that seemed to fix something. I'll try going back to unsigned here and see if it passes

i am pretty sure numpy random seed has to be <2**32, though

Reply:
Ha classic windows. Seems like tests do pass with uints so Ill approve